packages/auth/src/profile.ts

```diff
@@ -1,11 +1,11 @@
 import { resolveZoneFileToProfile } from '@stacks/profile';
 import { fetchPrivate } from '@stacks/common';
-import { StacksNetwork, StacksMainnet } from '@stacks/network';
+import { IStacksNetwork, StacksMainnet, StacksNetwork, StacksNetworkName } from '@stacks/network';
 
 export interface ProfileLookupOptions {
   username: string;
   zoneFileLookupURL?: string;
-  network?: StacksNetwork;
+  network?: StacksNetworkName | IStacksNetwork;
```

Comment:
Generally speaking elsewhere in Hiro's org typescript we're not following the `I` prefixed interface naming convention (keychain is as good as deprecated).

I see your comment about it, but I'd push to avoid this. It'll introduce a inconsistency, it isn't used in TS docs, and also odd when the type itself also begins with an `I` > `IIdentity`. I used to work on a project using this convention and haven't missed it.

There are bigger fish to fry, so your call, but my 2 euro cents.

Reply:
I went ahead and simplified this to just a single `class` now. The additional interface/class was making things too complicated here. Now we just have a class that is extended by `..Mainnet`/etc. and can also be used as a type — rather than having duplicate information with the interface+impl.